### DISCUSSION TITLE
Blockscout 2.0 release date?

### DISCUSSION THREAD

**Discussion by logginst** [blockscout/blockscout#7121] [Q&A] [answered]
Looking forward to the release of this. Has there been any rough timeline communicated about when this will be released? Is the code already available somewhere for those that are deploying themselves?

**Comment by ulyanas** [comment]
Hi, Tyler!

At this time, there is no set date for the release, but we are aiming to open-source the updated UI in Q2. 

We do offer early access to interested parties/chains. What is the best way to contact you?

We hope that this information is helpful, and we will continue to provide updates on the release timeline as they become available. Thank you for your interest in the product!

Blockscout team

**Reply by sheenhx** [reply]
Contact me please telegram : @Sheen_MetaX_Connect

**Reply by logginst** [reply]
Hi @ulyanas. That sounds great! Please email me at tyler@groma.com.

**Reply by ngdlong91** [reply]
Hi @ulyanas, may I have your contact for early access? Or my telegram: https://t.me/ngdlong91

**Reply by flaviusburca** [reply]
Hi @ulyanas, I am also interested in the new UI, Telegram: [@architect_zero](https://t.me/architect_zero). Can you please give me a sign ?

**Reply by ulyanas** [reply]
Greetings @logginst, @sheenhx, @ngdlong91, and @flaviusburca! We have granted you early access to the new UI repository. A representative from the Blockscout team will be in touch with you shortly.

**Reply by sheenhx** [reply]
Thanks , it is online for our first L3 zkEVM! And it looks nice! https://wannsee-explorer.mxc.com/

**Reply by thempp66** [reply]
Hi @ulyanas , Our project will be based on Blockscout 2.0 UI. So would you provide us early access please? My email is 914505102@qq.com. Thanks!

**Reply by Mojmir-R** [reply]
Hey @thempp66, thank you for your interest in the new version of Blockscout, can you please contact me on Telegram at Mojmir_R to discuss this further? Thank you!

**Reply by chrisciszak** [reply]
@ulyanas we are building a web3 gaming platform powered by OP Stack. Could you also give us early access to the new UI?

**Reply by Mojmir-R** [reply]
Hey @chrisciszak, please check my message right above yours. If you have already contacted me and I haven't replied yet, please send a reminder, thank you.

**Reply by CONGQUANG295** [reply]
Hi @ulyanas. I am also interested in the new UI. Contact me please telegram : [@congquang8292](https://t.me/congquang8292)

**Reply by Mojmir-R** [reply]
Hey @CONGQUANG295, please check my messages in this thread. If you are interested in early access, please send me a DM on Telegram at Mojmir_R

**Comment by TEAMDAOPSI2** [comment]
Hello, @ulyanas  I was wondering if it would be possible to request early access to the new UI repository. I believe this would be beneficial for my project and would greatly appreciate the opportunity. Thank you for your consideration.

**Reply by ulyanas** [reply]
@TEAMDAOPSI2  Could you please send us your contacts?

**Reply by TEAMDAOPSI2** [reply]
> @TEAMDAOPSI2 Could you please send us your contacts?

Hi you can contact me on my telegram sir https://t.me/TEAM_Psi_2 thank you.. sorry for the late reply

**Comment by luchenqun** [comment]
Hello, we are developing a new chain based on evmos and we need an EVM-compatible browser. We are very excited to see Blockscout 2.0 because it is exactly what we are looking for. Could you grant me access to the latest version of the Blockscout 2.0? My email and GitHub account are both [luchenqun@qq.com](mailto:luchenqun@qq.com).
@ulyanas

**Reply by Mojmir-R** [reply]
Hey @luchenqun, thank you for your interest in the new version of Blockscout, can you please contact me on Telegram at Mojmir_R to discuss this further? Thank you!

**Comment by xxxvik-xakerxxx** [comment]
I maintain several projects based on evm and use your product would love to try out the new version contact me on telegram @xxxvhxxx

**Reply by Mojmir-R** [reply]
Hey @xxxvik-xakerxxx, thank you for your interest in the new version of Blockscout, can you please contact me on Telegram at Mojmir_R to discuss this further? Thank you!

**Comment by ardier16** [comment]
Hello, @ulyanas! We're using Blockscout in our chain and would like to try out the new UI, it looks great! Could you please give me early access to the new UI repository?

Here are my contacts:
- Email: ardier16@gmail.com
- Telegram: https://t.me/ardier16

**Reply by Mojmir-R** [reply]
Hey @ardier16, thank you for your interest in the new version of Blockscout, can you please contact me on Telegram at Mojmir_R to discuss this further? Thank you!

**Comment by tuler** [comment]
I'd love to get early access to new UI as well. I'm developing tooling for [Cartesi](https://cartesi.io) rollups, and I'd like to add the possibility of running blockscout locally for dev. My telegram is @dtuler

**Reply by Mojmir-R** [reply]
Hey, @tuler, thank you for your interest in the new version of Blockscout, can you please contact me on Telegram at Mojmir_R to discuss this further? Thank you!

**Comment by avenbreaks** [comment]
hello, I'm from the [Gitshock Finance](https://gitshock.com) development team, L1 based on EVM and Cosmos, can we join for early access in the latest version of blockscout v5.1.5-beta with the latest look? previously we used version v5 for our 2 blockchain testnet and for this month we launched testnet phase 3. 

- https://scan.cartenz.works
- https://scan-circlechain.beaconchain.me

Contact :

- Email : dev-cosmware@gitshock.com
- Telegram : https://t.me/joywibowo
- Discord : joy-voister#4886


[Github Repo](https://github.com/gitshock-labs)

**Reply by Mojmir-R** [reply]
Hey @avenbreaks, thank you for your interest in the new version of Blockscout, can you please contact me on Telegram at Mojmir_R to discuss this further? Thank you!

**Reply by avenbreaks** [reply]
> Hey @avenbreaks, thank you for your interest in the new version of Blockscout, can you please contact me on Telegram at Mojmir_R to discuss this further? Thank you!

hello @Mojmir-R iam already DM you on discord, thankyou :)

**Comment by seonggwonyoon** [comment]
Our project is also using Blockscout as a base.
If possible, can you also do Frontend Early Access?

## Explorers
Bifrost Mainnet Explorer: https://explorer.mainnet.thebifrost.io/
Bifrost Testnet Explorer: https://explorer.testnet.thebifrost.io/

## Contact
Email Address: gwon@pi-lab.io

**Reply by Mojmir-R** [reply]
Hey, @seonggwonyoon, thank you for your interest in the new version of Blockscout, can you please contact me on Telegram at Mojmir_R to discuss this further? Thank you!

**Reply by nnlgsakib** [reply]
Hello. I am Sakib. I hope you are well.

I am employed as the Chief Developer of a Blockchain  called MIND Smart Chain.

As an explorer for our Blockchain we are interested in getting access to Blockscout 2.0

**Reply by Mojmir-R** [reply]
Hi Sakib, please contact me on Telegram and we can discuss more there.

**Comment by ekawibisono** [comment]
Hello, I'm very happy to see the new blockscout UI, can I use it?

**Reply by Mojmir-R** [reply]
Seems like you just contacted me on Telegram. If that is not the case, please reach out to Mojmir_R

**Reply by ekawibisono** [reply]
yep it's me, reply my DM, thanks

**Reply by ekawibisono** [reply]
Hello @Mojmir-R can you relpy my DM Telegram ? Username : @searchme_altacc

**Reply by Mojmir-R** [reply]
Replied. Sorry for the delay!

**Comment by aerielabio** [comment]
Hello, im very happy to see the new UI, is there anyway we can access it early ?

**Reply by Mojmir-R** [reply]
Hey @aerielabio, please DM me on Telegram at Mojmir_R

**Comment by G33kNoob** [comment]
hello any pricing for v2? or its still will be opensource

**Reply by Mojmir-R** [reply]
The new UI will be open-sourced in a few weeks.

**Comment by piersandro** [comment]
Hi, we would like to test the new UI on Quadrans Blockchain Testnet Explorer. Is it possibile to partecipate to the early access? Thank you.

**Reply by Mojmir-R** [reply]
Hey @piersandro, please ping me on Telegram at Mojmir_R and we can discuss there. Thanks

**Comment by Mojmir-R** [comment, marked as answer]
The new Blockscout UI is now officially open-sourced. More info here: https://discord.com/channels/973915550965174292/991979556242210936/1144341495642607686
